aws/required-permissions.md

```diff
@@ -96,8 +100,11 @@ Check out the templates directly for more information.
 | `logs:StopQuery` | Action | Allow Cloud One File Storage Security to stop a log query of the Lambda functions created by Cloud One File Storage Security. |
 | `logs:GetQueryResults` | Action | Allow Cloud One File Storage Security to get the log query results of the Lambda functions created by Cloud One File Storage Security. |
 | `logs:FilterLogEvents` | Action | Allow Cloud One File Storage Security to filter the log events of the Lambda functions created by Cloud One File Storage Security. |
+| `s3:GetBucketNotification` | Action | Allow Cloud One File Storage Security to get the notification configuration of a bucket. |
```

Comment:
How about using `the bucket to scan` across the doc.
Ref https://cloudone.trendmicro.com/docs/file-storage-security/arch-overview-aws/#S3BucketToScan
```suggestion
| `s3:GetBucketNotification` | Action | Allow Cloud One File Storage Security to get the notification configuration of the bucket to scan. |
```